### DISCUSSION TITLE
GitHub Actions: create PR instead of commits

### DISCUSSION THREAD

**Discussion by tinng81** [lowlighter/metrics#177] [Q&A] [answered]
Hi all, I've been a light user for sometimes and recently decided to invest more time into this. Incredible works, everyone!

For the GA, I can set up successfully for my profile repo and I notice that the GA is currently push each plugin outcome as a separate commit which could make my commit history clustered quickly. Albeit I can just clean it up, but I want to know if it is possible to resolve internally from the repo or the GA itself and possibly open a PR for contribution:

1. How to combine all plugin output from GA into one single commit?
2. Or better yet, is it possible to combine them as a single commit and create a pull request await for merging?

**Comment by lowlighter** [comment]
Hello!

Currently this not possible (unless you combine all plugin outputs in a single image by using multiple plugin inputs on the same step and you order content with `config_order`).

I can take a look at both tough, this seems feasible 🙂

**Comment by lowlighter** [comment, marked as answer]
`@master` now has #178 which makes it possible to configure output action (see [documentation](https://github.com/lowlighter/metrics/blob/master/source/plugins/core/README.md#-using-commits-pull-requests-or-manual-review-to-handle-metrics-output)), it'll be available on `@latest` in a few days 👍🏼
